applications/solvcon/prepare-solvcon-dev.sh

```diff
@@ -44,6 +44,10 @@ conda create -p ${SCSRC_WORKING}/venv-conda --no-default-packages -y python
 source activate ${SCSRC_WORKING}/venv-conda
 
 # prepare all packages to build SOLVCON
+source ${DEVENV_TOOL}/scripts/init
+devenv add ${DEVENV_FLAVOR}
+devenv use ${DEVENV_FLAVOR}
+VERSION=gmsh_3_0_6 devenv build gmsh
```

Comment:
Currently SOLVCON is still sticking to gmsh 3.0.6. We will also stick to this version until this issue is resolved https://github.com/solvcon/solvcon/issues/236 .